doc/REFERENCES.md

```diff
@@ -4,7 +4,7 @@ CAW currently uses GRCh37 by default. Support for GRCh38 is not fully working ye
 
 ## GRCg37
```

Comment:
Good catch, I'm waiting for some tests to finish on my machine, and I'll correct it.
Thanks